File: LatA/disk.cpp
#include <iostream>
#include <string>
using namespace std;
class disk{
    // Attribute
    private:
        string tipe;
        string kapasitas;
        int harga;
    public:
        // Constructor kosong
        disk(){}
        // Constructor
        disk(string type, string capacity, int price){
            this->tipe = type;
            this->kapasitas = capacity;
            this->harga = price;
        }
        // Get dan Set
        void setTipe(string type){
            this->tipe = type;
        }
        string getTipe(){
            return this->tipe;
        }
        void setKapasitas(int capacity){
            this->kapasitas = capacity;
        }
        string getKapasitas(){
            return this->kapasitas;
        }
        void setHarga(int price){
            this->harga = price;
        }
        int getHarga(){
            return this->harga;
        }
        // Fungsi untuk menampilkan isi dari attribute
        void printDisk(){
            cout << "Disk Type: " << this->tipe << endl;
            cout << "Disk Capacity: " << this->kapasitas << endl;
            cout << "Disk Price: " << this->harga << endl;
        }
        // Destructor
        ~disk(){}
};

File: LatB/Java/Disk.java
public class Disk{
    // Attribut untuk class Disk
    private String kapasitas;
    private int harga;
    private String tipe;
    // Constructor
    public Disk(String kapasitas, int harga, String tipe) {
        this.kapasitas = kapasitas;
        this.harga = harga;
        this.tipe = tipe;
    }
    // Constructor kosong
    Disk(){}
    // Get dan Set
    public String getKapasitas() {
        return kapasitas;
    }
    public void setKapasitas(String kapasitas) {
        this.kapasitas = kapasitas;
    }
    public int getHarga() {
        return harga;
    }
    public void setHarga(int harga) {
        this.harga = harga;
    }
    public String getTipe() {
        return tipe;
    }
    public void setTipe(String tipe) {
        this.tipe = tipe;
    }
    // Fungsi untuk menampilkan isi dari attribute class Disk
    public void printDisk(){
        System.out.println("Disk Type: " + this.tipe);
        System.out.println("Disk Capacity: " + this.kapasitas);
        System.out.println("Disk Price: " + this.harga);
    }
}